youtube_dl/extractor/pornorama.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..compat import compat_urllib_parse_unquote
+from ..utils import (
+    clean_html,
+    ExtractorError
+)
+
+
+class PornoramaIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?pornorama\.com/video(?P<id>[0-9]+)(?:.*)'
+    _TEST = {
+        'url': 'http://www.pornorama.com/video1416287/amy_my_neighbor_fucked',
```

Comment:
@dstftw  I found  the open site support request #7850 and it is as @Hrxn  says. So I thought I may give it a try. The example URL is in the #7850. If you feel it looks like copyrighted content, please reject this pull request.